components/script/dom/servoparser/mod.rs

```diff
@@ -351,14 +352,17 @@ impl ServoParser {
         self.document.set_current_parser(None);
 
         if self.pipeline.is_some() {
+            // Initial reflow.
             self.document.disarm_reflow_timeout();
             self.document.upcast::<Node>().dirty(NodeDamage::OtherNodeDamage);
             let window = self.document.window();
             window.reflow(ReflowGoal::ForDisplay, ReflowQueryType::NoQuery, ReflowReason::FirstLoad);
         }
 
-        // Step 3.
+        // Steps 3-12 are in another castle.
```

Comment:
Cute, but this neither explains what part of the spec this code implements nor where to find steps 3-12.